src/Plugin_Command.php

```diff
@@ -853,7 +882,7 @@ public function uninstall( $args, $assoc_args = array() ) {
 			$successes++;
 		}
 		if ( ! $this->chained_command ) {
-			Utils\report_batch_operation_results( 'plugin', 'uninstall', count( $args ), $successes, $errors );
+			Utils\report_batch_operation_results( 'plugin', 'uninstall', count( $plugins ), $successes, $errors );
```

Comment:
And this change isn't needed (if the `if ( count( $plugins ) < count( $args ) ) {` logic above is put in).